requirements.txt

```diff
@@ -0,0 +1,40 @@
+attrs==19.3.0
```

Comment:
This list of the dependencies is inaccurate.
All the dependencies necessary to run tests are defined in setup.py and can be installed by running 'pip install .[test]'

Reply:
Oh thanks for pointing it out. I'm used to the requirements.txt file so I manually installed those packages and did a 
```
pip freeze > requirements.txt
```
Will update the readme to point new users to install test dependencies.

Reply:
Addressed.